Zend/tests/bug40784.phpt

```diff
@@ -1,24 +0,0 @@
---TEST--
-Bug #40784 (Case sensivity in constructor's fallback)
---FILE--
-<?php
-
-class A {
-	  function A () { echo "I'm A\n"; }
-}
-
-class B extends A {
-  function __construct() {
-    parent::__construct();
-    parent::__constrUct();
-  }
-}
-
-$b = new B;
-
-echo "Done\n";
-?>
---EXPECTF--	
-I'm A
-I'm A
-Done
```

Comment:
There's a number of tests that are removed by this patch, but which still look relevant to me, like this one. Shouldn't they be kept, just with the additional deprecation notice? (Or for the lazy: Just remove E_DEPRECATED from error_reporting for the test.)